Modules/Loadable/Segmentations/Widgets/qMRMLSegmentsTableView.h

```diff
@@ -131,6 +131,9 @@ public slots:
   /// Segment's center is determined as the center of bounding box.
   void jumpSlices();
 
+  /// TODO
```

Comment:
Why is this a TODO?

Reply:
Because I was supposed to fill in the description before making the pull request. I'll fix that now.